### PR TITLE
SLI-1115 Add in-product links to learn more about focus on new code

### PR DESCRIPTION
**Pull request by damien-urruty-sonarsource**

### CODE REVIEW

File: src/main/java/org/sonarlint/intellij/cayc/FocusModeHelpLabel.kt [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+import com.intellij.ui.ContextHelpLabel
+import org.sonarlint.intellij.documentation.SonarLintDocumentation
+
+class FocusModeHelpLabel {
+    companion object {
+        @JvmStatic
+        fun create() =
+                ContextHelpLabel.createWithLink(null,
+                        "Deliver clean code by focusing on code that was recently modified",
+                        "Learn more about Clean as you Code", true) { BrowserUtil.browse(SonarLintDocumentation.FOCUS_ON_NEW_CODE_LINK) };
```
Comment (nquinquenel) [inline src/main/java/org/sonarlint/intellij/cayc/FocusModeHelpLabel.kt:32]:
```suggestion
                        "Learn more about Clean as You Code", true) { BrowserUtil.browse(SonarLintDocumentation.FOCUS_ON_NEW_CODE_LINK) };
```

Comment (nquinquenel) [reply src/main/java/org/sonarlint/intellij/cayc/FocusModeHelpLabel.kt:32]: Shouldn't we capitalize this?

**Review by nquinquenel — APPROVED** [review]
LGTM, just one comment

File: src/main/java/org/sonarlint/intellij/cayc/FocusModeHelpLabel.kt (hunk 1 above)
Comment (damien-urruty-sonarsource) [reply src/main/java/org/sonarlint/intellij/cayc/FocusModeHelpLabel.kt:32]: Yep, I can capitalize 'You'. Not the full sentence as from the guidelines [links](https://jetbrains.github.io/ui/text/capitalization/#sentence) should use sentence capitalization

**Pull request merged**